Makefile

```diff
@@ -43,11 +44,12 @@ LIBGTEST = /usr/lib/libgtest_main.so /usr/lib/libgtest.so
 LIBGMOCK = /usr/lib/libgmock_main.so /usr/lib/libgmock.so
 
 SUBDIRS = src/lib
+COPY_FILES = $(DISTDIR)/DejaVuSansMono.ttf
```

Comment:
This is no longer needed. I'm keep the fonts in the fonts dir and referencing them from there.